### PR TITLE
refactor/1583 - Update EDDF AIRAC to 2009

### LINKED ISSUE #1583: Update EDDF to newest AIRAC
AIRAC 2005 (23 APR 2020)

`PSA` NDB was replaced by `SPESA` fix.
-> new STARs and transitions

### PR DESCRIPTION
**Pull request by j-651**
Resolves #1583 
Supersedes #1623 

The purpose of this pull request is to begin work on EDDF AIRAC update.

Work by @matthewpow

### CODE REVIEW

File: assets/airports/eddf.json [hunk 1]
```diff
@@ -1108,7 +1108,7 @@
                 ["DF149", "MTR", "TOBAK", "APROX", "MARUN"]
             ]
         },
-        "MTR5C": {
+        "METRO5C": {
```
Comment (Fechulo) [inline assets/airports/eddf.json:1111]: The SID declaration should be the same as the ICAO name, so this should remain MTR5C. Same applies to Ried and Taunus SIDs as well.

File: assets/airports/eddf.json [hunk 2]
```diff
@@ -1911,36 +1911,36 @@
                 ["KERAX", "GED", "GEDSI", "MTR", "ETARU", "DF951", "IBLUS"]
             ]
         },
-        "KERAX3B": {
-            "icao": "KERAX3B",
-            "name": "Kerax Three Bravo",
+        "KERAX4B": {
```
Comment (Fechulo) [inline assets/airports/eddf.json:1914]: As with SIDs, STAR declarations need to have the same name as the ICAO name. so this should be KERA4B. Same applies to other STARs

Comment (matthewpow) [reply assets/airports/eddf.json:1914]: Will change both within the next few hours. Thanks!

**Comment by matthewpow** [comment]
Thanks, I'll get to work on those changes.

**Pull request merged**